### PR TITLE
flock: our second project!

### LINKED ISSUE #2: Add flock page
We should add a page about the flock project.

### PR DESCRIPTION
**Pull request by adamdecaf**
I noticed it wasn't listed today, so I threw something together. Figure it's good press for the projects too! 

Fixes: https://github.com/gofrs/website/issues/2

### CODE REVIEW

File: content/projects/flock/_index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: flock
+description: Thread-safe file locking in Go
+weight: 1
+---
+
+Our second project to adopt was [flock](https://github.com/gofrs/flock) (formerly known as `github.com/theckman/go-flock`). This project was donated to us from [@theckman](https://github.com/theckman) and we hapily adopted it because of its use in some projects across the community.
+
+Overall, the project seems stable and complete. There aren't a tons and tons of users, but some important projects. ([Import list](https://godoc.org/github.com/theckman/go-flock?importers))
```
Comment (theckman) [inline content/projects/flock/_index.md]: I would combine these two sentences:
>Based on its issue and PR history the project seems stable and complete, with some impactful projects making use of it.

**Review by theckman — CHANGES_REQUESTED** [review]
This looks good, with one piece of feedback about the second section. How do you feel about it?

File: content/projects/flock/_index.md (hunk 1 above)
Comment (adamdecaf) [reply content/projects/flock/_index.md]: Yea, that reads much better.

**Review by niaow — APPROVED** [review]
LGTM

**Comment by adamdecaf** [comment]
cc @theckman

**Comment by ldez** [comment]
I will merge this PR even if the website is dead.

I think I will work on a new website.

**Pull request merged**